mlflow/genai/scorers/base.py

```diff
@@ -105,6 +111,18 @@ def filter_string(self) -> str | None:
         """Get the filter string for this scorer."""
         return self._sampling_config.filter_string if self._sampling_config else None
 
+    @property
+    @experimental(version="3.3.0")
+    def status(self) -> ScorerStatus:
+        """Get the status of this scorer, using only the local state."""
+
+        if self._sampling_config is None:
+            return ScorerStatus.UNREGISTERED
+
+        if self.sample_rate is not None and self.sample_rate > 0:
+            return ScorerStatus.STARTED
+        return ScorerStatus.STOPPED
```

Comment:
```suggestion
        return ScorerStatus.STARTED if self.sample_rate > 0 else ScorerStatus.STOPPED
```

null is checked above